components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/api/impl/GroupImpl.java

```diff
@@ -50,6 +50,9 @@
 @SuppressWarnings("NonJaxWsWebServices")
 public class GroupImpl implements Group {
 
+    private static final String GROUP_CANNOT_NULL_MSG = "Group cannot be null.";
+    private static final String GROUP_NAME_INVALID_MSG = "Provided group name is invalid. Should be in minimum 3 " +
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/303/commits/051493db6a3fdb2e75cc7d1f48cb3ed407622312